### PR TITLE
feat: moveMode and drop to disable node #WIK-8685

### PR DESCRIPTION
**Pull request by Ashy6**

### CODE REVIEW

File: packages/mindmap/src/node.component.ts [hunk 1]
```diff
@@ -53,6 +53,7 @@ import { createEmptyNode, findPath, getChildrenCount } from './utils/mindmap';
 import { addSelectedMindmapElements, deleteSelectedMindmapElements, hasSelectedMindmapElement } from './utils/selected-elements';
 import { getNodeShapeByElement } from './utils/shape';
 import { ELEMENT_GROUP_TO_COMPONENT, MINDMAP_ELEMENT_TO_COMPONENT } from './utils/weak-maps';
+import { BaseCursorStatus } from 'packages/plait/src/interfaces';
```
Comment (huanhuanwa) [inline packages/mindmap/src/node.component.ts:56]: 引入路径不对

File: packages/plait/src/board/board.component.ts [hunk 2]
```diff
@@ -290,31 +294,37 @@ export class PlaitBoardComponent implements OnInit, AfterViewInit, OnDestroy {
...
+    moveEnd() {
+        this.move.isMoving = false;
+        this.move.x = 0;
+        this.move.y = 0;
     }
 
     // 拖拽模式
-    changeDragMode(cursorStatus: BaseCursorStatus) {
+    changeMoveMode(cursorStatus: BaseCursorStatus) {
+        if (cursorStatus === BaseCursorStatus.move) {
```
Comment (huanhuanwa) [inline packages/plait/src/board/board.component.ts:323]: 使用  @HostBinding('class') className = `plait-board-container plait-board-container-${this.board.cursor}`;

File: packages/mindmap/src/plugins/with-mindmap.ts [hunk 3]
```diff
@@ -82,7 +83,7 @@ export const withMindmap: PlaitPlugin = (board: PlaitBoard) => {
                 const mindmapComponent = MINDMAP_TO_COMPONENT.get(value);
                 const root = mindmapComponent?.root;
                 (root as any).eachNode((node: MindmapNode) => {
-                    if (hitMindmapNode(board, point, node)) {
+                    if (hitMindmapNode(board, point, node) && board.cursor !== BaseCursorStatus.move) {
```
Comment (huanhuanwa) [inline packages/mindmap/src/plugins/with-mindmap.ts:86]: board.cursor !== BaseCursorStatus.move 写到 hitMindmapNode 中吧

Comment (Ashy6) [reply packages/mindmap/src/plugins/with-mindmap.ts:86]: ok

File: packages/mindmap/src/node.component.ts (hunk 1 above)
Comment (Ashy6) [reply packages/mindmap/src/node.component.ts:56]: 现在改成了  from '@plait/layouts'

File: packages/plait/src/interfaces/board.ts [hunk 4]
```diff
@@ -47,8 +47,8 @@ export interface PlaitBoardOptions {
     allowClearBoard: boolean;
 }
 
-export interface DragMove {
-    isDragMoving: boolean;
+export interface Move {
```
Comment (huanhuanwa) [inline packages/plait/src/interfaces/board.ts:50]: PlaitBoardMove

**Review by huanhuanwa — APPROVED** [review]

**Pull request merged**